tests/functional/test_wheel.py

```diff
@@ -246,3 +246,22 @@ def test_legacy_wheels_are_not_confused_with_other_files(script, tmpdir, data):
     wheel_file_name = 'simplewheel-1.0-py%s-none-any.whl' % pyversion[0]
     wheel_file_path = script.scratch / wheel_file_name
     assert wheel_file_path in result.files_created, result.stdout
+
+
+def test_pip_option_save_wheel_name(script, data):
+    """
+    Test 'pip wheel ' success.
+    """
+    script.pip(
+        'wheel', '--no-index', '-f', data.find_links,
+        'require_simple==1.0',
+        '--save-wheel-name', 'wheelnames',
+    )
+
+    wheel_file_names = ['require_simple-1.0-py%s-none-any.whl' % pyversion[0],
+                        'simple-3.0-py%s-none-any.whl' % pyversion[0],
+                        ]
+    wheelnames_path = script.scratch_path / 'wheelnames'
+    wheelnames_file = open(wheelnames_path, 'r')
```

Comment:
This file is not closed. Please use the `open` context manager (`with open(...) as wheelnames_file`).